plugins/CoreHome/angularjs/report-export/reportexport.directive.js

```diff
@@ -30,7 +30,11 @@
 
                 scope.showUrl = false;
 
-                scope.getExportLink = function() {
+                scope.getExportLinkWithoutToken = function() {
+                    return scope.getExportLink(false);
+                }
+
+                scope.getExportLink = function(withToken = true) {
```

Comment:
can't use es6+ here unfortunately (this is also causing the angularjs tests to fail). after this is fixed, i'll merge (sorry i keep finding issues one at a time)

Reply:
yes, I figured yesterday this on the other PR, I will fix it quickly here as well